app/src/main/java/no/nordicsemi/android/wifi/provisioning/home/view/sections/DeviceSection.kt

```diff
@@ -91,7 +91,7 @@ private fun DeviceNotSelectedSection() {
     Column(horizontalAlignment = Alignment.CenterHorizontally) {
         Image(
             painter = painterResource(id = R.drawable.ic_nrf70),
-            contentDescription = stringResource(id = R.string.add_device),
+            contentDescription = null,
```

Comment:
Maybe we can add some string for sand-blind people?

Reply:
I understand, but the content description should describe the content. The image shows [nRF70 icon](https://github.com/NordicSemiconductor/Android-nRF-Wi-Fi-Provisioner/blob/main/app/src/main/res/drawable/ic_nrf70.png), not a "Add device" icon.
As I understand, that's a placeholder only for "no devices"?

Also, this icon should be placed in "drawables-xxhdpi"  folder, I guess? The default "drawable" is an alias for "mdpi", and it gets expanded 2 times on almost all phones.

Reply:
Yes, the string was obviously wrong so it should be sth more accurate.
I don't know about the icon. If xxhdpi solves the problem then it should be moved there.